public/styles.css

```
@@ -1,3 +1,44 @@
+*{
+    box-sizing: border-box;
+}
+
+html,body{
+    height: 100%;
+    width: 100%;
+    background-color:#faffffff ;
+}
+.background{
+    height: 100%;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    
+}
+.vs{
+    width: 50%;
+}
+.enemy{
+    width:"100%";
```

```suggestion
    width: 100%;
```
ダブルクォート要らないです :pray: